labs/8.md

```diff
@@ -1,9 +1,103 @@
 
-Thanks for attending this hands on lab. By now you have completed the following activities.
+## Create a Namespace
+To start with we will create a namespace for the *reverse-words* application to be deployed.
 
-* Explored the environment
-* Deployed KubeFed
-* Flattened out your configuration files
-* Deployed 2 applications
-* Etc...
+~~~sh
+oc create ns --context cluster1 reverse-words
+oc project --context cluster1 reverse-words
+~~~
+
+Now that the namespace exists deploy the deployment and service objects.
+~~~sh
+oc create --context cluster1 -f 01-service.yaml
+oc create --context cluster1 -f 02-deployment.yaml
+oc expose --context cluster1 svc/reverse-words
+oc expose svc/reverse-words
+~~~
+
+Verify the application is running.
+~~~sh
+oc get all
+NAME                                 READY   STATUS    RESTARTS   AGE
+pod/reverse-words-7fc48b5bd4-h9qlv   1/1     Running   0          14s
+
+NAME                    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/reverse-words   ClusterIP   172.31.2.137   <none>        8080/TCP   16s
+
+NAME                            READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/reverse-words   1/1     1            1           14s
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/reverse-words-7fc48b5bd4   1         1         1       14s
+
+NAME                                     HOST/PORT                                               PATH   SERVICES        PORT   TERMINATION   WILDCARD
+route.route.openshift.io/reverse-words   reverse-words-reverse-words.apps.east-1.sysdeseng.com          reverse-words   8080                 None
+~~~
+
+Using curl test the application using the route displayed above.
+~~~sh
+curl http://reverse-words-reverse-words.apps.east-1.sysdeseng.com/ -X POST -d '{"word": "PALC"}'
+{"reverse_word":"CLAP"}
+~~~
+
+## Federate Objects
+Now that the application has been deployed and is operation; we will federate the object using the kubefedctl.
+
+~~~sh
+kubefedctl federate namespace reverse-words
+~~~
+
+Ensure that the namespace has been federated
+
+~~~sh
+for i in cluster1 cluster2 cluster3; do oc get ns reverse-words --context $i; done
+NAME            STATUS   AGE
+reverse-words   Active   3m52s
+NAME            STATUS   AGE
+reverse-words   Active   41s
+NAME            STATUS   AGE
+reverse-words   Active   42s
+~~~
+
+Now that the namespace has been federated let's federate the deployment and service so that the objects will be populated to the other clusters.
+
+~~~sh
+kubefedctl federate deployments.apps reverse-words --context cluster1
+kubefedctl federate service reverse-words --context cluster1
```

Comment:
I believe the correct command would be:

kubefedctl federate service -n reverse-words --host-cluster-context=cluster1